openslides/utils/autoupdate.py

```diff
@@ -133,50 +136,80 @@ def send_data(message):
                     {'text': json.dumps(output)})
 
 
-def inform_changed_data(instance, information=None):
+def inform_changed_data(instances, information=None):
     """
     Informs the autoupdate system and the caching system about the creation or
     update of an element.
+
+    The argument instances can be one instance or an interable over instances.
     """
-    try:
-        root_instance = instance.get_root_rest_element()
-    except AttributeError:
-        # Instance has no method get_root_rest_element. Just ignore it.
-        pass
-    else:
-        collection_element = CollectionElement.from_instance(
-            root_instance,
-            information=information)
-        # If currently there is an open database transaction, then the
-        # send_autoupdate function is only called, when the transaction is
-        # commited. If there is currently no transaction, then the function
-        # is called immediately.
-        transaction.on_commit(lambda: send_autoupdate(collection_element))
+    root_instances = set()
+    if not isinstance(instances, Iterable):
+        # Make surce instance is an iterable
+        instances = (instances, )
+    for instance in instances:
+        try:
+            root_instances.add(instance.get_root_rest_element())
+        except AttributeError:
+            # Instance has no method get_root_rest_element. Just ignore it.
+            pass
+
+    # Generates an collection element list for each root_instance.
+    collection_elements = CollectionElementList()
+    for root_instance in root_instances:
+        collection_elements.append(
+            CollectionElement.from_instance(
+                root_instance,
+                information=information))
+    # If currently there is an open database transaction, then the
+    # send_autoupdate function is only called, when the transaction is
+    # commited. If there is currently no transaction, then the function
+    # is called immediately.
+    transaction.on_commit(lambda: send_autoupdate(collection_elements))
 
 
-def inform_deleted_data(collection_string, id, information=None):
+def inform_deleted_data(*args, information=None):
     """
     Informs the autoupdate system and the caching system about the deletion of
-    an element.
+    elements.
+
+    The function has to be called with the attributes collection_string and id.
+    Multible elements can be used. For example:
+
+    inform_deleted_data('motions/motion', 1, 'assignments/assignment', 5)
+
+    The argument information is added to each collection element.
     """
-    collection_element = CollectionElement.from_values(
-        collection_string=collection_string,
-        id=id,
-        deleted=True,
-        information=information)
+    if len(args) % 2 or not args:
```

Comment:
Hm. This might be `if not args or not len(args) % 2:` because you want to catch zero args or an odd number of args.